os/include/sys/syscall.h

```diff
@@ -99,13 +99,13 @@
 #define SYS_sched_yield                (CONFIG_SYS_RESERVED + 12)
 #define SYS_set_errno                  (CONFIG_SYS_RESERVED + 13)
 
-#ifdef CONFIG_SMP
 #define SYS_sched_getaffinity          (CONFIG_SYS_RESERVED + 14)
 #define SYS_sched_setaffinity          (CONFIG_SYS_RESERVED + 15)
+#ifdef CONFIG_SMP
```

Comment:
In the same point of my request, we should enable all syscall API regardless of related configs. But the config is disabled, then we can do above among the options. 
We can do this with new PR.